NeoFlux/Controllers/ContractController.cs

```diff
@@ -17,6 +18,16 @@ namespace NeoFlux.Controllers
     [Route("api/[controller]")]
     public class ContractController : BaseController
     {
+        public static IConfiguration Configuration { set; get; }  

```

Comment:
I think you should use one of the factories to hold this, instead of spreading this out through controllers.